extensions/wikia/ArticleComments/classes/ArticleCommentsAjax.class.php

```diff
@@ -28,7 +28,7 @@ class ArticleCommentsAjax {
 	 * @return String -- json-ized array
 	 */
 	static public function axSave() {
-		global $wgRequest, $wgUser, $wgTitle;
+		global $wgRequest, $wgUser;
 
 		$articleId = $wgRequest->getVal( 'article', false );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "article" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)